install/kubernetes/helm/istio/charts/pilot/templates/config.yaml

```diff
@@ -0,0 +1,28 @@
+{{ define "config.yaml.tpl" }}
```

Comment:
This should go in the mixer chart ( it's not specific to pilot )

Reply:
No, mixer does not define these CRDs so it is inappropriate to submit pilot CRs in mixer helm chart. How would isolated mixer install work?

Reply:
Use 'pilot.enabled' to exclude - if mixer can run without pilot.

Reply:
This is not about enablement, but the fact that CRDs are defined in pilot.